app/assets/javascripts/item_selector/item_selector_limit_message.js

```diff
@@ -0,0 +1,55 @@
+var itemSelectorLimitMessage = (function() {
+
+  var limitReachedMessage = function(limit) {
+    return [
+      '<div id="max-items-reached" class="alert alert-danger" role="alert">',
```

Comment:
I wonder if this message should live on the rails side of things, in case we need it in a non-javascript situation.